pkg/cmd/server/start/master_args.go

```diff
@@ -199,7 +199,11 @@ func (args MasterArgs) BuildSerializeableMasterConfig() (*configapi.MasterConfig
 			Latest: args.ImageFormatArgs.ImageTemplate.Latest,
 		},
 
-		ProjectRequestConfig: configapi.ProjectRequestConfig{},
+		ProjectConfig: configapi.ProjectConfig{
+			DefaultNodeSelector:    args.ProjectNodeSelector,
```

Comment:
Yes, removed the command line arg. @pravisankar, advanced options like default node selector are only available via config